sdk/informaticadatamanagement/Azure.ResourceManager.InformaticaDataManagement/api/Azure.ResourceManager.InformaticaDataManagement.net8.0.cs

```diff
@@ -59,8 +62,6 @@ protected InformaticaOrganizationResource() { }
         public virtual Azure.Response<Azure.ResourceManager.InformaticaDataManagement.Models.InformaticaServerlessRuntimeResourceList> GetAllServerlessRuntimes(System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
         public virtual System.Threading.Tasks.Task<Azure.Response<Azure.ResourceManager.InformaticaDataManagement.Models.InformaticaServerlessRuntimeResourceList>> GetAllServerlessRuntimesAsync(System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
         public virtual System.Threading.Tasks.Task<Azure.Response<Azure.ResourceManager.InformaticaDataManagement.InformaticaOrganizationResource>> GetAsync(System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
-        public virtual Azure.Response<Azure.ResourceManager.InformaticaDataManagement.InformaticaServerlessRuntimeResource> GetInformaticaServerlessRuntime(string serverlessRuntimeName, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
```

Comment:
Is this expected?

Reply:
As I can see, this operation should belong to collection, not resource. Because this operation is used to get `InformaticaServerlessRuntimeResource`, and it should not belong to `InformaticaServerlessRuntimeResource` itself.

Reply:
This operation is duplicated with https://github.com/Azure/azure-sdk-for-net/blob/40c338b4c0037c63e8925052a2329beadf18add2/sdk/informaticadatamanagement/Azure.ResourceManager.InformaticaDataManagement/src/Generated/Extensions/MockableInformaticaDataManagementArmClient.cs#L43-L59.
I think the newly generated code is more reasonable.